R/mod_introJS_utils_text.R

```diff
@@ -84,6 +83,31 @@ mm_steps <-
     position = c(rep("left", 2))
   )
 
+# Package Explorer
+pe_steps <- 
+  data.frame(
+    element = c("#file_tree", "#file_editor","#comments_for_se"),
```

Comment:
Since these are defined within a module, they really should be using the namespace of that module.

Reply:
done